picturae_csv_create.py

```diff
@@ -7,24 +7,21 @@
 import picturae_config
 import logging
 from taxon_parse_utils import *
-from picturae_import_utils import *
+from gen_import_utils import *
 from string_utils import *
 from importer import Importer
 from sql_csv_utils import SqlCsvTools
 from specify_db import SpecifyDb
 import picdb_config
 
-# creating a batch uuid
-batch_uuid = uuid4()
-
-
 starting_time_stamp = datetime.now()
 
 
 class CsvCreatePicturae(Importer):
-    def __init__(self, date_string):
+    def __init__(self, date_string, logger):
```

Comment:
takes logger as arg to allow logger to be passed from importer to csv_creator. Can be discussed as per comment on last PR.

Reply:
pass logging level down, not the logger